Ginger/Ginger/ApplicationModelsLib/POMModels/PomElementsPage.xaml.cs

```diff
@@ -881,7 +882,19 @@ private void HighlightElementClicked(object sender, RoutedEventArgs e)
 
             if (mSelectedElement != null)
             {
+
+                try
+                {
                     mWinExplorer.HighLightElement(mSelectedElement, true, mPOM?.MappedUIElements);
+                }
+                catch (Exception ex)
+                {
+                    if(ex is NoSuchElementException)
+                    {
+                        Reporter.ToUser(eUserMsgKey.ElementNotFound);
+                        Reporter.ToLog(eLogLevel.INFO, ex.Message, ex);
```

Comment:
Log exception for all type of exception and level should be error Reporter.ToLog(eLogLevel.ERROR, ex.Message, ex);

Show user pop up only for ElementNotFoundException.